### PR TITLE
PRMT-3488

### PR DESCRIPTION
**Pull request by martin-nhs**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -45,6 +43,7 @@ dependencies {
     implementation 'org.bouncycastle:bcpkix-jdk15on:1.70'
     implementation 'com.google.code.gson:gson:2.10.1'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
```
Comment (AndyFlintNHS) [inline build.gradle]: Remove duplicate line

File: src/main/java/uk/nhs/prm/e2etests/configuration/AsyncConfiguration.java [hunk 2]
```diff
@@ -15,9 +15,9 @@ public class AsyncConfiguration implements AsyncConfigurer {
     @Override
     public Executor getAsyncExecutor() {
         ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
-        executor.setCorePoolSize(7);
-        executor.setMaxPoolSize(42);
-        executor.setQueueCapacity(11);
+        executor.setCorePoolSize(10);
```
Comment (AndyFlintNHS) [inline src/main/java/uk/nhs/prm/e2etests/configuration/AsyncConfiguration.java]: This class is no longer used, can we remove it?

File: src/test/java/uk/nhs/prm/e2etests/performance/RepoInPerformanceTest.java [hunk 3]
```diff
@@ -51,13 +51,13 @@ class RepoInPerformanceTest {
     TestConfiguration config;
 
     @Autowired
-    TrackerDb trackerDb;
+    TransferTrackerService transferTrackerService;
 
     @Autowired
-    TransferCompleteQueue transferCompleteQueue;
+    EhrTransferServiceTransferCompleteOQ transferCompleteQueue;
```
Comment (martin-nhs) [inline src/test/java/uk/nhs/prm/e2etests/performance/RepoInPerformanceTest.java:57]: Address name.

File: src/test/java/uk/nhs/prm/e2etests/ActiveRoleArnTest.java [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
         // Given
-        final ExampleAssumedRoleArn exampleAssumedRoleArn = new ExampleAssumedRoleArn(EXAMPLE_ARN);
+        final ActiveRoleArn activeRoleArn = new ActiveRoleArn(EXAMPLE_ARN);
 
         // Then
-        assertThat(exampleAssumedRoleArn.getTargetArn()).isEqualTo("arn:aws:sts::123456789154:role/RepoAdmin");
+        assertThat(activeRoleArn.getTargetArn()).isEqualTo("arn:aws:sts::123456789154:role/RepoAdmin");
     }
 
     // TODO: PRMT-3488 Test name no longer matches logic!
```
Comment (AndyFlintNHS) [inline src/test/java/uk/nhs/prm/e2etests/ActiveRoleArnTest.java:21]: Do we need to rename these tests? This TODO needs addressing for this ticket

File: src/test/java/uk/nhs/prm/e2etests/live_technical_test/day1/ChangeOfGPMessageReceivedTest.java [hunk 5]
```diff
@@ -73,14 +72,14 @@ void shouldHaveReceivedSingleSuspensionChangeOfGpMessageRelatedToTestPatient() {
                 System.out.println("Patient suspended status is:" + pdsResponse.getIsSuspended());
 
                 System.out.println("Checking patient status on HL7 PDs lookup - see gp2gp messenger logs for insights");
-                gp2GpMessengerClient.getPdsRecordViaHlv7(nhsNumber);
+                gp2GpMessengerService.getPdsRecordViaHlv7(nhsNumber);
```
Comment (AndyFlintNHS) [inline src/test/java/uk/nhs/prm/e2etests/live_technical_test/day1/ChangeOfGPMessageReceivedTest.java]: Address this 'hlv7', should just be 'hl7'

**Review by AndyFlintNHS — CHANGES_REQUESTED** [review]

**Pull request merged**